tests/models/test_gpu.py

```diff
@@ -49,7 +49,7 @@ def test_multi_gpu_none_backend(tmpdir):
     tpipes.run_model_test(trainer_options, model)
 
 
-@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
 @pytest.mark.parametrize('gpus', [1, [0], [1]])
 def test_single_gpu_model(tmpdir, gpus):
```

Comment:
Is the test description wrong then? It say "test_single_gpu"

Reply:
it's a test running on a single gpu, but on gpu index "1", therefore this requires a machine with at least 2 gpus.